models/pull.go

```diff
@@ -185,7 +185,7 @@ func (pr *PullRequest) GetDefaultMergeMessage() string {
 		log.Error("Cannot load issue %d for PR id %d: Error: %v", pr.IssueID, pr.ID, err)
 		return ""
 	}
-	return fmt.Sprintf("Merge pull request '%s' (#%d) from %s/%s into %s", pr.Issue.Title, pr.Issue.Index, pr.MustHeadUserName(), pr.HeadBranch, pr.BaseBranch)
+	return fmt.Sprintf("Merge pull request '%s' (#%d) from %s/%s/%s into %s", pr.Issue.Title, pr.Issue.Index, pr.MustHeadUserName(), pr.HeadRepo.Name, pr.HeadBranch, pr.BaseBranch)
```

Comment:
If prefer it if the delimiter between branch and repo was a colon or a space not a slash.

```suggestion
	return fmt.Sprintf("Merge pull request '%s' (#%d) from %s/%s:%s into %s", pr.Issue.Title, pr.Issue.Index, pr.MustHeadUserName(), pr.HeadRepo.Name, pr.HeadBranch, pr.BaseBranch)
```

That way it's clear that it's not a repo name.

Reply:
@zeripath I have made the change.